### PR TITLE
add numeric password and imeOptions

### PR DESCRIPTION
**Pull request by Stallinator**

### CODE REVIEW

File: demo/src/main/java/de/check24/demo/features/password/numeric/ComposableNumericPasswordActivity.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        }
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun NumberPassword() {
```
Comment (Tautellini) [inline demo/src/main/java/de/check24/demo/features/password/numeric/ComposableNumericPasswordActivity.kt:52]: Not sure what we decided, but the demo project should probably also have working Previews.

Comment (Stallinator) [reply demo/src/main/java/de/check24/demo/features/password/numeric/ComposableNumericPasswordActivity.kt:52]: absolutly right. well spotted

File: demo/src/main/java/de/check24/demo/features/password/visibility/ComposablePasswordVisibilityToggleActivity.kt [hunk 2]
```diff
@@ -15,10 +15,7 @@ import androidx.compose.material.icons.filled.VisibilityOff
 import androidx.compose.runtime.*
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.input.KeyboardType
-import androidx.compose.ui.text.input.PasswordVisualTransformation
-import androidx.compose.ui.text.input.TextFieldValue
-import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.input.*
```
Comment (Tautellini) [inline demo/src/main/java/de/check24/demo/features/password/visibility/ComposablePasswordVisibilityToggleActivity.kt:18]: Not super important for this repo I guess, but maybe change your project settings to not have wildcard imports. There are 2 good reasons not to use them:

First, you may get problems with naming conflicts without noticing it, because the WildCard import may be used, even if you dont want it to.
Secondly, this will raise compile-time (even if just a little I guess)

Comment (Stallinator) [reply demo/src/main/java/de/check24/demo/features/password/visibility/ComposablePasswordVisibilityToggleActivity.kt:18]: saw that as well, no idea why its in there, will configure :)

**Comment by Stallinator** [comment]
pls dont merge. for some reason imeOption done does not close keyboard anymore

**Comment by Stallinator** [comment]
works as expected now

File: code/text/TextViewPasswordVisibility.kt [hunk 3]
```diff
@@ -1,8 +1,10 @@
+@OptIn(ExperimentalComposeUiApi::class)
```
Comment (Darius3001) [inline code/text/TextViewPasswordVisibility.kt:1]: Why is this necessary? Im not to familiar with this thats why im asking

**Review by Darius3001 — APPROVED** [review]
looks fine to me
